utils/cautils/offline.go

```diff
@@ -274,7 +274,7 @@ func (c *OfflineCA) SSHSign(req *api.SSHSignRequest) (*api.SSHSignResponse, erro
 		ValidAfter:  req.ValidAfter,
 		ValidBefore: req.ValidBefore,
 	}
-	cert, err := c.authority.SignSSH(publicKey, signOpts, opts...)
+	cert, err := c.authority.SignSSH(context.Background(), publicKey, signOpts, opts...)
```

Comment:
@maraino is `context.Background()` correct here (and similar elsewhere)? I didn't really know what to use. The other option would be a fresh context.

Reply:
This is ok.